### PR TITLE
Images

### PR DESCRIPTION
**Pull request by stevemar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,33 +131,40 @@ Click **View Credentials** and save the **username** and **password** values.
...
+As part of the service, you will be assigned the following values, which you will need to save, they are accessible through your user account settings:
 
-As part of the service, you will be assigned the following values, which you will need to save:
 - Account SID
 - Authentication Token
-- Twilio supplied phone number (where all texts will originate from)
 
-By default, you will only be able to send SMS text messages to the phone number you
-provided to create your account. To send messages to other numbers, you need to add
-them at [https://www.twilio.com/console/phone_numbers/verified](https://www.twilio.com/console/phone_numbers/verified).
```
Comment (stevemar) [inline README.md:147]: this link didn't work for me if i didn't have any phone numbers yet.

File: README.md [hunk 2]
```diff
@@ -170,6 +180,15 @@ $ cp config.new.js config.js
 $ node run.js
 ```
 
+### Troubleshooting
+
+You may need to install dependencies before running the application, to do
```
Comment (rhagarty) [inline README.md]: I think we need to add this in the line above run.js. This will ALWAYS be needed.

**Review by rhagarty — CHANGES_REQUESTED** [review]
Just the one suggestion. Nice pics, btw

**Comment by stevemar** [comment]
@rhagarty addressed your comments, refer to latest change set

**Review by rhagarty — APPROVED** [review]

**Pull request merged**